### DISCUSSION TITLE
Can "included minutes" be used on macOS instances?

### DISCUSSION THREAD

**Discussion by sejr** [community/community#24914] [Actions]
<p>The title pretty much says it all. I actually just purchased a Team membership and was curious if the 10,000 included Actions minutes can be used building/testing iOS applications.</p>
<p>Thanks!</p>

**Comment by chrispat** [comment, marked as answer]
<p>Yes they can but they are not 1:1.  The included minutes are based on the Linux price of $.008 per minute and macOS is 10x that at $.08.  So every minute of macOS will be “charged” as 10 minutes to your included quantity.</p>

**Comment by sejr** [comment]
<p>Is this documented somewhere?</p>

**Comment by chrispat** [comment]
<p>No becuase we are not charging yet and the limits are not yet being enforced.</p>

**Comment by sejr** [comment]
<p>Good to know. Thanks!</p>

**Comment by tonyarnold** [comment]
<p>Sorry to resurrect this thread, but my gosh that 10x multiplier is bad value. </p>
<p>Does GitHub have plans to lower this cost somehow?</p>

**Comment by skydivedan** [comment]
<p>Yes, this seems absolutely terrible. Why in the world would it cost <strong>10 times</strong> the amount? Is there a shortage on Mac Mini’s or something?</p>

**Comment by ajsirota** [comment]
<p>It’s probably because they legally can’t virtualize macOS, like they can Windows or Linux.</p>
